src/client/remote_sequential.py

```diff
@@ -85,3 +86,11 @@ def inference_session(self, **kwargs) -> RemoteSequentialInferenceSession:
 
     def extra_repr(self) -> str:
         return f"modules={self.sequence_manager.block_uids[0]}..{self.sequence_manager.block_uids[-1]}"
+
+
+class RemoteTransformerBlock(RemoteSequential):
+    """Single transformer block hosted by swarm"""
```

Comment:
```suggestion
    """Single transformer block hosted by swarm
    
    This class is deprecated and kept for backward compatibility.
    It will be removed soon in favor of using ``RemoteSequential`` directly.
    """
```